javascript-sdk/package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "lint:debug": "eslint src/sdk.ts --debug",
-    "test": "mocha --require ts-node/register tests/**/*.test.ts"
+    "test": "tests/setup-and-run-tests.sh"
```

Comment:
Lets leave the this call as is, and instead run `tests/setup-and-run-tests.sh` instead of `npm test` in the github workflow